app/src/main/java/ca/bc/gov/vaxcheck/ui/scanner/BarcodeScannerFragment.kt

```diff
@@ -127,9 +128,10 @@ class BarcodeScannerFragment : Fragment(R.layout.fragment_barcode_scanner), Scan
         }
 
         binding.txtPrivacyPolicy.setSpannableLink {
-            val uri = Uri.parse("https://www2.gov.bc.ca/gov/content/covid-19/vaccine/proof/businesses#app-privacy-policy")
-            val intent = Intent(Intent.ACTION_VIEW, uri)
-            requireContext().startActivity(intent)
+            val uri = Uri.parse(getString(R.string.url_privacy_policy))
+            val action = BarcodeScannerFragmentDirections
```

Comment:
No need to pass from this screen, as we are going to get it from the required screen.

Reply:
changes done.